index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Desmond Owusu</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <div class="background"></div>
+    <header class="Header">
+      <div class="headerContent">
+        <a class="Logo" href="#">Desmond Owusu</a>
+        <nav>
+          <ul class="navItems">
+            <li class="navItem"><a href="#">Portfolio</a></li>
+            <li class="navItem"><a href="#">About</a></li>
+            <li class="navItem"><a href="#">Contact</a></li>
+          </ul>
+        </nav>
+        <img src="images/menu.png" alt="" class="Menu" />
+      </div>
+    </header>
+    <main class="main">
+      <section class="mainContent">
+        <h1 class="description">
+          I am Desmond, <br />
+          glad to see <br />
+          you!
+        </h1>
+        <p class="aboutMe">
+          I'm a software developer! I can help you build a product, feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hestiate to contact
+          me.
+        </p>
+        <p class="letsConnect">LET'S CONNECT</p>
+        <div class="networkContent">
+          <ul class="links">
+            <li><a href="https://twitter.com/DesmondOwusuDev"><img src=" ./images/twitter.svg" alt="twitter"></a></li>
+            <li><a href="https://www.linkedin.com/in/desmond-owusu-ansah-09274a223/"><img src=" ./images/Linkedin-icon.svg" alt="Linkedin"></a></li>
+            <li><a href="https://github.com/Owusu-Desmond"><img src=" ./images/github.svg" alt="github"></a></li>
+            <li><a href="#"><img src=" ./images/M-icon.svg" alt="M-icon"></a> </li>
+            <li><a href="#"><img src=" ./images/hand-raise.svg" alt="handraise icon"></a></li>
+          </ul>
+        </div>
+      </section>
```

Comment:
- [ ] It is required that your template should match the one provided by Microverse. Check the [templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) and Ensure that you `fonts`, `colours` and `icons` are almost similar to the template reference.

1. **Your design**
![image](https://user-images.githubusercontent.com/80612925/170543972-07b4f503-ee73-446b-81ef-94c14e4a280f.png) 
2. **Figma desing**
![image](https://user-images.githubusercontent.com/80612925/170544645-1a77f62d-ed6c-42cb-923b-776d69966ed7.png)

Reply:
**[REQUIRED CHANGES]**

- As required by the previous reviewer, please kindly add the missing color to the `LET'S CONNECT`
- Add the correct spacing between the social media icons.
- increase the font size of the paragraph to match the Figma design
- Remove the  second `<br>` from your h1 to make it seat properly